ui/src/message_popup/author/new_question_page.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import QuestionPage from './question_page.jsx';
+import {allArchivedQuestions} from './archived_questions.js';
+import {allQuestions} from '../questions.js';
+import * as Api from '../../helpers/api.js';
+
+
+export default React.createClass({
+  displayName: 'NewQuestionPage',
+
+  getInitialState(){
```

Comment:
After this PR, let's talk out or pair to factor out the "loading questions from the server" bit of this component, `new_question_page` and `questions_page`.  We can make what's called a "higher-order component" that does this and then passes along the questions to child components.  That will help simplify and split out the loading code into one component.